source/guides/network/creating-a-private-test-net.rst

```diff
@@ -171,6 +147,7 @@ On the other hand, you can share securely the public and address of your account
 
 Now that you have your node running an account with some |privatenetworkcurrency|, we recommend you to surf the |sitename|. In this portal, you can find everything you need to know about |codename|'s features and :ref:`self-paced guides <blog-categories>` on how to use the **software development kits**.
```

Comment:
Now that you have your node running an account with some |privatenetworkcurrency|, we recommend you to surf the |sitename|. In this portal, you can find detailed information about |codename|'s features and :ref:`self-paced guides <blog-categories>` on how to use the **software development kits**.